includes/class-user.php

```diff
@@ -42,6 +42,9 @@ public static function get( $user = 0 ) {
 		if ( empty( $user ) ) {
```

Comment:
You could optimize this block by doing inline check instead of two blocks

```php
if ( empty( $user ) || ! is_a( $user, 'WP_User') ) {
    return null;
}
```